### PR TITLE
prometheus-postgres-exporter - Allow readiness timeout to be configurable via config

### PR DESCRIPTION
**Pull request by Narayana108**
#### What this PR does / why we need it:

**What this PR does**
Adds `readinessProbe.timeoutSeconds` to be configurable via values.yaml file. 

**Why**
 We are running slow SQL queries which take about 3 min to complete on postgres exporter and the pod keeps on restarting on start up, as it exceeds the default readiness timeout. By increasing this value we can run our slow queries without issues.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by zanhsieh — DISMISSED** [review]

**Comment by zanhsieh** [comment]
@Narayana108 
Could you sign DCO please? Thank you.

**Comment by Narayana108** [comment]
@zanhsieh Done :)

File: charts/prometheus-postgres-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "0.9.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 2.2.0
+version: 2.2.1
```
Comment (monotek) [inline charts/prometheus-postgres-exporter/Chart.yaml:5]: New feature should raise minor version.

File: charts/prometheus-postgres-exporter/templates/deployment.yaml [hunk 2]
```diff
@@ -101,6 +101,7 @@ spec:
               port: http
           readinessProbe:
             initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+            timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
```
Comment (monotek) [inline charts/prometheus-postgres-exporter/templates/deployment.yaml:104]: The k8s default of one second should be in the values.yaml

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-postgres-exporter/templates/deployment.yaml (hunk 2 above)
Comment (Narayana108) [reply charts/prometheus-postgres-exporter/templates/deployment.yaml:104]: Sorry im not to sure what you mean by this. Can one by default change the k8s readiness timeoutSeconds ?

Comment (monotek) [reply charts/prometheus-postgres-exporter/templates/deployment.yaml:104]: You're using a variable .Values.readinessProbe.timeoutSeconds which is not defined in values.yaml.
Therefore variable would be empty by default.
The Kubernetes standard value for timeoutSeconds is 1 second.
Therefore it should be in the values.yaml like this.

**Review by monotek — CHANGES_REQUESTED** [review]

**Comment by Narayana108** [comment]
@monotek I have added the readiness timeout to values.yaml.

**Review by monotek — CHANGES_REQUESTED** [review]
may i ask you to add the same for livenessprobe too?
so both probes can be configured the same way.

**Comment by Narayana108** [comment]
@monotek Done

**Review by zanhsieh — APPROVED** [review]

**Review by monotek — APPROVED** [review]

**Pull request merged**